docs/kafka/nodejs-kafka/README.adoc

```diff
@@ -197,29 +194,31 @@ endif::[]
 The Node.js application in this quick start uses a Kafka topic called `countries` to produce and consume messages. In this task, you'll create the topic in your Kafka instance.
 
 .Prerequisites
-* You've created a Kafka instance in {product-kafka} and the instance is in the *Ready* state.
+* You've created a Kafka instance in {product-long-kafka} and the instance is in the *Ready* state.
 
 .Procedure
-. In the {product-kafka} web console, go to *Streams for Apache Kafka* > *Kafka Instances* and click the name of the Kafka instance that you want to add a topic to.
-. Click *Create topic* and follow the guided steps to define the topic details. Click *Next* to complete each step and click *Finish* to complete the setup.
+. In the {product-kafka} web console, select *Kafka Instances* and then click the name of the Kafka instance that you want to add a topic to.
+. Select the *Topics* tab, click *Create topic*, and follow the guided steps to define the topic details, as shown in the figure.
```

Comment:
I notice that we've got some steps like this (throughout our documentation) where we role three steps into 1. I'd like to break this step up into two parts for increased readability:

- Select the **Topics** tab.
- Click **Create topic** and follow the guided steps to define the topic details as shown in the following figure.

Reply:
Yep, I agree. I'll update just these two guides in this PR and make a note that we need to create an issue to update others.